PointOfDispensingApp/config.js

```diff
@@ -0,0 +1,10 @@
+//file name is 'config.js'
+
+export default {
+    vaccinationDecisionState:
+    {
+        UNDETERMINED: 0,
```

Comment:
This is a great candidate for state management once it's implemented

Reply:
Noted